tamr_toolbox/notifications/emails.py

```diff
@@ -83,10 +85,18 @@ def send_email(
 
     try:
         context = ssl.create_default_context()
-        # send message
-        with smtplib.SMTP_SSL(smtp_server, smtp_port, context=context) as server:
-            server.login(sender_address, sender_password)
-            response = server.sendmail(sender_address, recipient_addresses, msg)
+        # select cryptographic protocol
+        if use_tls:
+            server = smtplib.SMTP(smtp_server, smtp_port)
+            server.starttls(context=context)
+        else:
+            server = smtplib.SMTP_SSL(smtp_server, smtp_port, context=context)
```

Comment:
You are no longer using a contextmanager via `with` as you were before. Why not?

Reply:
Is there a simple way to use `with` when you have two different functions with different parameters? I tried to reduce the number of duplicate lines of code by removing `with`. 

Alternative:
if use_tls:
   with smtplib.SMTP as server...
   # repetitive code here
else:
   with smtplin.SMTP_SSL as server...
   # repetitive code also here

Reply:
You could try:
```python
with smtplib.SMTP if use_tls else smtplib.SMTP_SSL as server:
   # repetitive code here
```

Reply:
Thanks!